### ISSUE TITLE
Lorem.sentences() returning words instead of sentences

### ISSUE THREAD

**Issue by phillippbertram** [drager/faker#14]
Hi!

I just saw that the `Lorem.sentences` method does not output sentences but words.

**Original**
```dart
List<String> sentences(numberOfSentences) {
    return Iterable<int>.generate(numberOfSentences).map((_) => word()).toList();
}
```

**Expected**
```dart
List<String> sentences(num numberOfSentences) {
    return Iterable<int>.generate(numberOfSentences).map((_) => sentence()).toList();
}
```

✌️

**Comment by drager** [comment]
Thanks for reporting! Feel free to send a pull request that addresses that

**Comment by Windstalker** [comment]
@drager I see you fixed the Issue on latest master.

**Comment by drager** [comment]
@Windstalker Ah yes! I will make a release soon that includes that fix!

**Comment by drager** [comment]
@phillippbertram @Windstalker Released in [1.2.0](https://pub.dev/packages/faker/versions/1.2.0)
